Backpack-SwiftUI/NavigationBar/Classes/BPKNavigationView.swift

```diff
@@ -0,0 +1,140 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+public struct BPKNavigationView<Content: View>: View {
+    let title: String?
+    let style: Style
+    let leadingItems: [Self.ToolbarItem]
+    let trailingItems: [Self.ToolbarItem]
+    let content: () -> Content
+    
+    public init(
+        title: String? = nil,
+        leadingItems: [Self.ToolbarItem] = [],
+        trailingItems: [Self.ToolbarItem] = [],
+        style: Style = .default(.automatic),
+        content: @escaping () -> Content
+    ) {
+        self.title = title
+        self.style = style
+        self.leadingItems = leadingItems
+        self.trailingItems = trailingItems
+        self.content = content
+        
+        let appearance = UINavigationBarAppearance()
+        appearance.configureWithOpaqueBackground()
+        appearance.titleTextAttributes = [.foregroundColor: BPKColor.statusSuccessFillColor.value]
+        appearance.largeTitleTextAttributes = [.foregroundColor: BPKColor.statusSuccessFillColor.value]
+        UINavigationBar.appearance().standardAppearance = appearance
+        UINavigationBar.appearance().scrollEdgeAppearance = appearance
```

Comment:
Question on these lines, wouldn't all navigation bars in the app change after we set this?

Reply:
My understanding is NO, it should not change other navbars in the app. Testing in the Backpack Sample App, this does not change other navbars. It would only affect navigation bars that has not been configured before hand. Besides this, in the Skyscanner app we currently have 0 native navigation bars (to my knowledge), they're all custom views placed on top of the view controllers.

If after releasing this change we see it messes with the app in any way we should make a release removing this change!